rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/util/EvaluatorUtil.java

```diff
@@ -163,23 +165,31 @@ private static void reportInternalError(Throwable e, String task, LanguageClient
         var msg = new ShowMessageRequestParams();
         msg.setMessage(title);
         msg.setType(MessageType.Error);
-        msg.setActions(Arrays.asList(new MessageActionItem("Report on GitHub"), new MessageActionItem("Ignore")));
+        final String ReportLabel = "Report on GitHub";
```

Comment:
It all looks good to me, but maybe this whole function can benefit from some extracted functions.

1. calculate the reason
2. calculate the stack trace
3. open the GH issue
4. copy to clipboard.

Reply:
It could indeed.